Source/Core/Core/IOS/ES/Views.cpp

```diff
@@ -164,8 +149,8 @@ IPCReply ESDevice::GetV0TicketFromView(const IOCtlVRequest& request)
   {
     return IPCReply(ES_EINVAL);
   }
-  return IPCReply(GetV0TicketFromView(Memory::GetPointer(request.in_vectors[0].address),
-                                      Memory::GetPointer(request.io_vectors[0].address)));
+  return IPCReply(GetTicketFromView(Memory::GetPointer(request.in_vectors[0].address),
```

Comment:
Yet another theoretical concern, but I think this command is only supposed to return v0 tickets (unlike GetTicketFromView).